torch/csrc/jit/script/init.cpp

```diff
@@ -316,6 +316,16 @@ std::shared_ptr<SugaredValue> toSugaredValue(
   } else if (py::isinstance<py::module>(obj)) {
     return std::make_shared<PythonModuleValue>(obj);
   }
+
```

Comment:
nit: Can we put this below _find_builtin? In this order every time someone calls `torch.foo` we are making a check to `_try_compile_weak_script`. I think weak script functions will be rarer than builtins so perf will be better if we do it in the other order.